benchmarks/result_analyzer.py

```diff
@@ -255,8 +259,8 @@ def parse_args(args=None):
 
   parser.add_argument(
```

Comment:
Do we still need this flag?

Reply:
It's useful to have it if we're processing old json files that have no timestamps in them.
It's true though that eventually this flag should probably go away.